api/pkg/apps/iam/client_interface.go

```diff
@@ -19,9 +19,9 @@ type Interface interface {
 }
 
 type PartyListOptions struct {
-	PartyTypeID string            `json:"partyTypeId" bson:"partyTypeId"`
-	SearchParam string            `json:"searchParam" bson:"searchParam"`
-	Attributes  map[string]string `json:"attributes"`
+	PartyTypeID string
+	SearchParam string
+	Attributes  map[string]string
```

Comment:
Removed json/bson tags because we are not serializing/deserializing this struct using json